### DISCUSSION TITLE
Draw Arrows or Connectors between cells

### DISCUSSION THREAD

**Discussion by precisemath** [TanStack/table#4732] [Q&A]
Microsoft Excel has a feature where we can trace the antecedents/precedents of a cell by using their trace function.
We are trying to implement something similar in our project. Is it possible to connect 2 cells with an arrow/connector?

**Comment by RossCoeH** [comment]
I am working on that too, to create something similar to Gantt chart. 
Currently needing a hint on how to measure height and width of body section. 

My plan is to overlay a transparent svg over one column with linepaths onto it.
 I need to find window position of  top left corner of first visible cell, and bottom right visible cell of same column to define the overlay svg. Not solved yet, but looking through table state is the likely method.
